tests/integration/conftest.py

```diff
@@ -41,6 +42,7 @@
                                  "JOB_NAME": "JOB_NAME",
                                  "MOLECULE_TEST_REPO": "MOLECULE_TEST_REPO",
                                  "MOLECULE_SCENARIO_NAME": "MOLECULE_SCENARIO_NAME",
+                                 "MOLECULE_GIT_COMMIT": "Unknown",
```

Comment:
Setting this to "Unknown" prevents ZigZag from trying to build "Failure Links" which is the default state we want for fixtures.